apps/examples/testcase/le_tc/kernel/tc_libc_math.c

```diff
@@ -141,6 +375,60 @@ static void tc_libc_math_fdim(void)
 }
 
 /**
+ * @fn                   :tc_libc_math_fdimf
+ * @brief                :Returns the positive  difference max(x-y,0),  between thier arguments
+ * @Scenario             :Returns the positive  difference max(x-y,0),  between their arguments
+ * API's covered         :fdimf
+ * Preconditions         :None
+ * Postconditions        :None
+ * @return               :void
+ */
+static void tc_libc_math_fdimf(void)
+{
+	const float in_val1[10] = { ZERO, -ZERO, VAL1, -VAL1, VAL2, -VAL2, INFINITY, -INFINITY, NAN, VAL1 };
+	const float in_val2[10] = { -INFINITY, INFINITY, -VAL2, VAL2, -VAL1, VAL1, -ZERO, ZERO, NAN, NAN };
+	const float sol_val[10] = { INFINITY, ZERO, 6755399441055744.0, ZERO, 6755399441055744.0, ZERO, INFINITY, ZERO, NAN, NAN };
+	float ret_val[10];
+	int fdimf_idx;
+
+	for (fdimf_idx = 0; fdimf_idx < 9; fdimf_idx++) {
+		ret_val[fdimf_idx] = fdimf(in_val1[fdimf_idx], in_val2[fdimf_idx]);
+		if (!(isnan(sol_val[fdimf_idx]) && isnan(ret_val[fdimf_idx]))) {
+			TC_ASSERT_LEQ("fdimf", fabsf(sol_val[fdimf_idx] - ret_val[fdimf_idx]), FLT_EPSILON);
+		}
+	}
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+ * @fn                   :tc_libc_math_fdiml
+ * @brief                :Returns the positive  difference max(x-y,0),  between thier arguments
+ * @Scenario             :Returns the positive  difference max(x-y,0),  between their arguments
+ * API's covered         :fdiml
+ * Preconditions         :None
+ * Postconditions        :None
+ * @return               :void
+ */
+static void tc_libc_math_fdiml(void)
+{
+	const long double in_val1[10] = { ZERO, -ZERO, VAL1, -VAL1, VAL2, -VAL2, INFINITY, -INFINITY, NAN, VAL1 };
+	const long double in_val2[10] = { -INFINITY, INFINITY, -VAL2, VAL2, -VAL1, VAL1, -ZERO, ZERO, NAN, NAN };
+	const long double sol_val[10] = { INFINITY, ZERO, 6755399441055744.0, ZERO, 6755399441055744.0, ZERO, INFINITY, ZERO, NAN, NAN };
+	long double ret_val[10];
+	int fdiml_idx;
+
+	for (fdiml_idx = 0; fdiml_idx < 9; fdiml_idx++) {
```

Comment:
@sunghan-chang I made changes as per your comments. kindly review it https://github.com/Samsung/TizenRT/pull/321/commits/8f482040ef7d65dadddba8b4b385d36e7112c902